docs/release_notes/0.5.0.md

```diff
@@ -0,0 +1,18 @@
+# Release 0.5.0
```

Comment:
We should review this before we merge. It's possible we can cut an RC first also.

Reply:
Perhaps better to discuss outside of this PR.

Reply:
https://github.com/weaveworks/eksctl/issues/1268